source/patterns/@aws-solutions-constructs/core/lib/cloudfront-distribution-defaults.ts

```diff
@@ -42,11 +42,12 @@ export function DefaultCloudFrontWebDistributionForApiGatewayProps(apiEndPoint:
 
 export function DefaultCloudFrontWebDistributionForS3Props(sourceBucket: s3.IBucket, loggingBucket: s3.Bucket | undefined,
   setHttpSecurityHeaders: boolean,
+  originPath?: string | '/',
```

Comment:
Since '/' is a string, I don't think adding | '/' has any effect - it gives a false appearance of type checking.

I could send '\' or '|'

If they want '/' they should send undefined (not send an argument) - that would give them the default.

Reply:
This is a good point. I'll change this. Thank you.